.github/workflows/playwright.yml

```diff
@@ -36,9 +36,11 @@ jobs:
           npx playwright install
 
       - name: Run Playwright tests
-        run: npm run test:${{ inputs.environment }}
-        env:
-          E2E_TEST_ENV: ${{ inputs.environment }}
+        run: |
+          commandenv=":${{ inputs.environment != null && inputs.environment || 'stage' }}"
+          npm run test:$commandenv
+        env:          
+          E2E_TEST_ENV: ${{ inputs.environment != null && inputs.environment || 'stage' }}
```

Comment:
If I'm reading it correctly, I think this line isn't used?

```suggestion
```

Reply:
its used, essentially what this line does in yml is use inputs.environment if its provided (not null), else if null use 'stage'

Reply:
Ah got it, the env var is used in the test itself, not in the command. I assume it should be posible to change it to the following?

```
        run: npm run test:$E2E_TEST_ENV
        env:          
          E2E_TEST_ENV: ${{ inputs.environment != null && inputs.environment || 'stage' }}
```